src/VideoSubscriberAccount/VSAccountMetadataRequest.cs

```diff
@@ -21,7 +23,10 @@ public partial class VSAccountMetadataRequest {
 				return VSAccountProviderAuthenticationSchemeExtensions.GetValues (SupportedAuthenticationSchemesString);
 			}
 			set {
-				SupportedAuthenticationSchemesString = value?.GetConstants ();
+				if (value.GetConstants () is not null)
+					SupportedAuthenticationSchemesString = value.GetConstants ()!;
+				else
+					ObjCRuntime.ThrowHelper.ThrowArgumentNullException (nameof (value));
```

Comment:
IMHO using a temporary variable here would be better:

```suggestion
				var constants = value.GetConstants ();
				if (constants is null)
					ObjCRuntime.ThrowHelper.ThrowArgumentNullException (nameof (value));
				SupportedAuthenticationSchemesString = constants;
```

Reply:
Okay changing this to the proposed code!
However, the `value.GetConstants ();` returns a NSString?[]?
so I went with this code and added a `!` to the last line after constants. Internally, these methods throw an exception if it does find a null item inside the constants array and I believe this is what was recommended last time a similar thing happened